src/slack/conversation_test.go

```diff
@@ -42,3 +42,16 @@ func TestConversation_UpdateConversation(t *testing.T) {
 		}
 	}
 }
+
+func TestConversation_ClearConversation(t *testing.T) {
+	var c = newConversation()
+	c.data = map[string][]string{
+		"key1": {"1", "2", "3", "4"},
+		"key2": {"1", "2", "3", "4"},
+	}
+	var isCleared = c.ClearConversation("key1", c)
```

Comment:
A better way to test this would be to see if it can have a race condition:
```
for i := 0; i < 10; i++ {
    go func() {
        c.ClearConversation("key1", c)
     }
}
```
then run the test suite with `go test -race ./...`